spec/datasets/djornl/definitions.yaml

```diff
@@ -21,20 +21,31 @@ definitions:
       pattern: ^(\S+__){3}(\S+)$
     _from:
       type: string
-      title: Gene ID
+      title: Node ID
+      format: regex
+      pattern: ^\S{2,}.*$
```

Comment:
Adding in some extra constraints on IDs to ensure that there is some meaningful data in them